hivemind/averaging/allreduce.py

```diff
@@ -205,28 +262,32 @@ async def rpc_aggregate_part(
     ) -> AsyncIterator[averaging_pb2.AveragingData]:
         """a peer sends us a part of his tensor; we should average it with other peers and return the difference"""
         request: averaging_pb2.AveragingData = await anext(stream)
```

Comment:
Please handle `self.sender_timeout` for this first part as well.

Reply:
done